tests/e2e/cairo1/test_cairo1_testing.py

```diff
@@ -16,3 +18,28 @@ def test_no_tests_found(protostar: ProtostarFixture):
     result = protostar(["test-cairo1", "tests"])
 
     assert "No test cases found" in result
+
+
+def test_failing_tests(protostar: ProtostarFixture, copy_fixture: CopyFixture):
+    copy_fixture("cairo1_project", "./cairo1_project")
+    template_test_files = Path("cairo1_project/tests").glob("**/*")
+    for template_test_file in template_test_files:
```

Comment:
Perhaps this should use a version of protostar fixture which initializes a cairo1 project? Is it possible?

Reply:
@MaksymilianDemitraszek was against this approach afaik

Reply:
It is a cairo1 project, I just removed the template tests as they don't check what we need here. If you feel like we should include them anyway, please let me know but I think we're good without 'em.